src/nvim/api/win_config.c

```diff
@@ -415,10 +617,26 @@ static bool parse_float_relative(String relative, FloatRelative *out)
   return true;
 }
 
+static bool parse_float_split(String split, WinSplit *out)
```

Comment:
This function name is a bit inaccurate.

Reply:
Yeah, agreed. Maybe `parse_config_split` or something would be better? I just copied the naming scheme of the other properties since I wasn't sure what to go with.

Reply:
Renamed to `parse_config_split`, but feel free to reopen this if you'd prefer something else.